code/game/turfs/closed/wall/mineral_walls.dm

```diff
@@ -256,7 +256,7 @@
 	sheet_type = /obj/item/stack/sheet/mineral/titanium
 	hardness = 40 //6 on moh's scale
 	smoothing_flags = SMOOTH_BITMASK | SMOOTH_DIAGONAL_CORNERS
-	smoothing_groups = list(SMOOTH_GROUP_WALLS, SMOOTH_GROUP_TITANIUM_WALLS, SMOOTH_GROUP_CLOSED_TURFS) // SKYRAT EDIT CHANGE - Sorting them because /tg/ forgot to.
+	smoothing_groups = list(SMOOTH_GROUP_TITANIUM_WALLS, SMOOTH_GROUP_CLOSED_TURFS) // SKYRAT EDIT CHANGE - Sorting them because /tg/ forgot to.
```

Comment:
Might want to bring that up to /tg/ too, then, since it's a non-modular change. Only reason it's got a modular comment is because it was causing troubles with CI for us, because the groups weren't ordered.

Reply:
This is not an issue upstream as their windows do not smooth with walls